sixpack.js

```diff
@@ -199,7 +199,21 @@
         } else {
             var httpModule = url.startsWith('https') ? 'https' : 'http';
             var http = eval('require')(httpModule); // using eval to skip webpack bundling and warnings
-            var req = http.get(url, { headers: { 'Cookie': cookie } }, function(res) {
+
+            const parsedUrl = new URL(url);
+            const options = {
+                protocol: parsedUrl.protocol,
+                host: parsedUrl.host,
+                port: parsedUrl.port,
+                hostname: parsedUrl.hostname,
+                search: parsedUrl.search,
+                pathname: parsedUrl.pathname,
+                path: parsedUrl.pathname + parsedUrl.search, 
+                href: parsedUrl.href,
```

Comment:
You could use just `hostname`, `port` and `path`. Could you make a test?

Reply:
Yeah. It worked. I tested:

1. With port forwarding:
```bash
kl pf argos-api-759766dd8-29kgh 8081
```
Url definition:
```bash
cd src/jusbrasil/deadpool
export ARGOS_URL="http://www.local.jusbrasil.com.br:8081"
yarn dev
```

2. Public argos url
```bash
cd src/jusbrasil/deadpool
export ARGOS_URL="https://endpoints.jusbrasil.com.br/argos"
yarn dev
```

Reply:
I'd add `protocol` too